it.cuteworks.pacmanlogviewer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "it.cuteworks.pacmanlogviewer",
+    "rename-icon": "plv",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "plv",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=/var/log/pacman.log:ro"
```

Comment:
that can't be a file.

Reply:
not even the log file of the package manager?